src/main/scala/nl.knaw.dans.easy.stage/dataset/Remarks.scala

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (C) 2015 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.stage.dataset
+
+import java.io.{ File, FileNotFoundException }
+import java.nio.charset.StandardCharsets
+import java.nio.file.{ Files, NoSuchFileException, Paths }
+
+import nl.knaw.dans.lib.error._
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+import nl.knaw.dans.lib.string._
+import org.apache.commons.lang.BooleanUtils
+
+import scala.util.Try
+import scala.xml.{ Elem, XML }
+
+case class Remarks(bagitDir: File) extends DebugEnhancedLogging {
```

Comment:
Would it make sense to call this `Agreements`, just as we have `EMD` and `AMD`?

Reply:
The object sits at a cross-road between `agreements.xml` and `message-from-depositor.txt` on the input side and AMD/EMD component at the other size. So I abandoned `Agreements` as class name.

Now that I'm looking again into it, perhaps `DepositorInfo` is a better choice: the metadata directory that contains both input files.